src/components/VoteButtons.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div class="row rounded-borders vote-buttons items-center">
+    <q-btn
+      icon="arrow_upward"
+      :color="userVote === 'up' ? 'primary' : undefined"
+      flat
+    />
+    <div class="text-subtitle2 q-px-sm">
+      {{ voteTotal }}
+    </div>
+    <q-btn
+      icon="arrow_downward"
+      :color="userVote === 'down' ? 'primary' : undefined"
+      flat
+    />
+  </div>
+</template>
+
```

Comment:
It could be worth implementing highlighting the downvote button if clicked in this PR.

Reply:
If `userVote` is  `'up'` or `'down'` the highlighting works. I'll get that value change working when I start communicating with the server